feat(cli): add new flag --max-file-size to control the max file size (#6670)
Checkmarx/kics: docs/commands.md

                                      example: 'e69890e6-fce5-461d-98ad-cb98318dfc96,4728cd65-a20c-49da-8b31-9c08b423e4db'
      --input-data string             path to query input data files
  -b, --libraries-path string         path to directory with libraries (default "./assets/libraries")

>       --max-file-size int             max file size that is allowed to be scanned, in MB (default 5)

Reviewer: ```suggestion
      --max-file-size int             max file size permitted for scanning, in MB (default 5)
```
